src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC } from 'react';
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../../types';
+import { TableColumnsNames } from '../../types/TableColumnsNames';
+import { PersonLink } from '../PersonLink';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const columns = Object.values(TableColumnsNames);
+
+  const { slug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {columns.map(column => (
+            <th key={column}>{column}</th>
+          ))}
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const { sex, born, died, motherName, fatherName } = person;
+
+          const foundMother = people.find(mother => mother.name === motherName);
+          const foundFather = people.find(father => father.name === fatherName);
+
+          return (
+            <tr
+              data-cy="person"
+              key={person.slug}
```

Comment:
Ensure that `person.slug` is unique for each person. Using a non-unique key can lead to rendering issues. If `slug` is not guaranteed to be unique, consider using a combination of unique identifiers or indices.